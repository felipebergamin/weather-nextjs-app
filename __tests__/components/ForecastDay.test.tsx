import { render } from '@testing-library/react';

import App from '../../src/components/ForecastDay';

describe('testing app', () => {
  it('should match snapshot', () => {
    const { baseElement } = render(
      <App
        data={{
          dt: 1602354646,
          main: {
            feels_like: 65.7,
            temp: 60.1,
          },
          weather: [
            {
              icon: '123',
            },
          ],
        }}
      />,
    );

    expect(baseElement).toMatchSnapshot();
  });
});
