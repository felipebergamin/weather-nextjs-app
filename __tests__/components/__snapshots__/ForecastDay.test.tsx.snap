// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing app should match snapshot 1`] = `
<body>
  <div>
    <div
      class="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
      style="margin: 0.25rem;"
    >
      <div
        class="MuiCardContent-root"
        style="display: flex; flex-direction: column;"
      >
        <div
          class="MuiCardMedia-root"
          style="background-image: url(undefined/123@4x.png); width: 100%; height: 120px; background-size: contain;"
        />
        <p
          class="MuiTypography-root MuiTypography-body1"
        >
          <strong>
            today at 3:30 PM
          </strong>
        </p>
        <span
          class="MuiTypography-root MuiTypography-caption"
        >
          Temperature: 
          60.1
           ℉
        </span>
        <span
          class="MuiTypography-root MuiTypography-caption"
        >
          Feels like: 
          65.7
           ℉
        </span>
        <span
          class="MuiTypography-root MuiTypography-caption"
        >
          Humidity: 
          60.1
          %
        </span>
      </div>
    </div>
  </div>
</body>
`;
